<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTestLog</class>
 <widget class="QDialog" name="DialogTestLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测试记录</string>
  </property>
  <widget class="QTableView" name="tableView1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>581</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
  </widget>
  <widget class="QTableView" name="tableView2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>971</width>
     <height>581</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCount">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>600</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCount">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>600</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>测试总数：0      通过数量:  0    通过率： 0.00%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
